src/middlewares/validators/updateUserPayload.js

```diff
@@ -0,0 +1,21 @@
+const Joi = require('joi');
+
+const updateUserPayloadValidator = (req, res, next) => {
+  const schema = Joi.object({
+    name: Joi.string().alphanum().min(2).max(45).trim(true),
+    surname: Joi.string().alphanum().min(2).max(45).trim(true),
+    middleName: Joi.string().alphanum().min(2).max(45).trim(true),
+    password: Joi.string().min(8).max(100),
```

Comment:
Create please separate endpoint to update password, which accepts oldPassword and newPassword, compares old passwords and then update it. Create please also seperate ticket to add 3 new endpoints: createAdmin (it has the same structure as createUser, but role will be "Admin"), blockUser (accepts user id in req.params and sets userStatus as "Blocked"), deleteUser (accepts user id in req.params and sets userStatus as "Deleted")